src/tree/CosmosDBAccountWizard/CosmosDBAccountApiStep.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardStep, IAzureUserInput, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { ICosmosDBWizardContext } from './ICosmosDBWizardContext';
+import { Experience, DBAccountKind } from '../../constants';
+
+export class CosmosDBAccountApiStep extends AzureWizardStep<ICosmosDBWizardContext> {
+    public async prompt(wizardContext: ICosmosDBWizardContext, ui: IAzureUserInput): Promise<ICosmosDBWizardContext> {
+        const picks: IAzureQuickPickItem<string>[] = [
+            { label: Experience.MongoDB, description: '', data: DBAccountKind.MongoDB },
+            { label: Experience.DocumentDB, description: '', data: DBAccountKind.GlobalDocumentDB },
```

Comment:
Shouldn't this be SQL now?  With a description that it used to be docdb?

Reply:
I think the whole renaming business is very confusing and I don't want to deal with it 🙃

FIled an issue to address separately: #415